netconfig.md

```diff
@@ -1,8 +1,8 @@
 # Network configuration
 
-In order to participate in the Spacemesh network and to earn mining awards your computer firewall (and in some cases your home network router) so the Spacemesh App is able to accept incoming Internet connections on `port 7153` for both `TCP` and `UDP`.
+In order to participate in the Spacemesh network and to earn mining awards your computer firewall (and in some cases your home network router) needs to be configured, so the Spacemesh App is able to accept incoming Internet connections on `port 7153` for both `TCP` and `UDP`.
 
-You only need to configure your network if you are planning to run a Miner and not if you plan to use the App in the custom wallet only mode.
+You only need to configure your network if you are planning to run a full p2p node and not if you plan to use the App in the custom wallet-only mode.
 
 ---
 
```

Comment:
We should be consistent along the guide about the order of operating systems.
Here we have Win 10, OSX, Linux, but in https://testnet.spacemesh.io/#/no_sleep for example we have OSX before Win 10.